docs/contributing/documentation-guidelines/index.md

```diff
@@ -19,3 +21,36 @@ In terms of style, you should refer to the [Google developer documentation style
 - [Use sentence case](https://developers.google.com/style/capitalization) for document titles and section headings.
 - [Use descriptive link text](https://developers.google.com/style/link-text).
 - [Write short sentences](https://developers.google.com/style/translation#write-short,-clear,-and-precise-sentences) that are easy to understand and translate.
+
+## How to preview your modification on Autoware Documentation
+
+There are two ways to preview your modification on Autoware Documentation.
+
+### 1. Use Github Actions workflow in `autowarefoundation/autoware-documentation` repository
+
+You can use Github Actions in `autowarefoundation/autoware-documentation` repository to deploy your branch.
+
+1. Create a PR of your branch to the repository.
+2. Add a `documentation` label from the sidebar.
+3. Wait for a couple of minutes, and the `github-actions` will notify the URL for your branch's preview.
```

Comment:
- Add `bot` to be specific.
- I feel `pull request` is better than `branch` here.

```suggestion
3. Wait for a couple of minutes, and the `github-actions` bot will notify the URL for your pull request's preview.
```

Reply:
Resolved https://github.com/autowarefoundation/autoware-documentation/commit/ff49a125b82682e0d062b355a2a4a3258451dcd9